html/canvas/tools/yaml-new/drawing-images-to-the-canvas.yaml

```diff
@@ -437,8 +437,8 @@
         resolve(xhr.response);
       };
     });
-    promise.then(function(response) {
-      createImageBitmap(response).then(bitmap => {
+    return promise.then(function(response) {
+      return createImageBitmap(response).then(bitmap => {
```

Comment:
I think a better fix would be to use await for the promise above. It looks like these end up in
`promise_test(async` so that should be fine. Maybe also `const` the `xhr`  while there...

Reply:
Oh, so the `async` keyword is there in the generated function body. I guess I'll make everything nice then.